### ISSUE TITLE
No way to add 2nd user?

### ISSUE THREAD

**Issue by francisbarton** [MarkXA/TwitDuck#6]
Hi

I believe you should be able to add multiple users in tweetdeck/twitduck, but I can't find any way to achieve this. Clicking on "Accounts" just brings up the one account I'm logged in with, with no apparent way to add a 2nd account.

**Comment by MarkXA** [comment]
Yes, they made it nice and intuitive now. Basically you have to log in to Tweetdeck with the second account, use the pulldown in the accounts section to add @FirstAccount to "team @SecondAccount", then log back in with the first account. Obvious, really.

**Comment by francisbarton** [comment]
I'll give that a try - thank you!
